lib/styles/trackInputModality.js

```diff
@@ -45,7 +45,7 @@ function setUpEventHandlers() {
       return el.msMatchesSelector;
     }
 
-    logging.error("Couldn't find any matchesSelector method on document.body.");
+    // logging.error("Couldn't find any matchesSelector method on document.body.");
   })();
 
   const disableFocusRingByDefault = function() {
```

Comment:
I think we could remove `":not([modality=keyboard])"` from the line 52. It seems that [it was copied from alice/modality](https://github.com/alice/modality/blob/master/src/keyboard-modality.js#L29) which [updates this attribute](https://github.com/alice/modality/blob/master/src/keyboard-modality.js#L65) but we don't do that. Instead, we [update `globalThemeState.inputModality`](https://github.com/learningequality/kolibri-design-system/blob/v0.2.x/lib/styles/trackInputModality.js#L101) and use that information in our components.

Is it okay to remove it, or were there any plans to use it?

Reply:
yes - should be fine to remote it

Reply:
yes - should be fine to remote it